### PR TITLE
Optimize behat tests 

### PR DESCRIPTION
**Pull request by pablothedude**
This PR has some optimizations in order to help debug failing features.

- Make behat replay logic more verbose
- Change the deprecated twig localizeddate function
- Run all Behat features in isolation

Todo:
- [x] add dedicated replay test

### CODE REVIEW

File: stepup/tests/behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -44,24 +45,64 @@ class FeatureContext implements Context
...
+
+        // Push config
+        echo "Push Middleware config\n";
+        self::execCommand("./fixtures/middleware-push-config.sh");
+        self::execCommand("./fixtures/middleware-push-whitelist.sh");
+        self::execCommand("./fixtures/middleware-push-institution.sh");
+
+        // Write base setup for initializing features
+        echo "Dump empty setup to mysql file\n";
+        self::execCommand("mysqldump -h mariadb -u root -psecret --single-transaction --databases middleware_test gateway_test > setup.sql");
```
Comment (johanib) [inline stepup/tests/behat/features/bootstrap/FeatureContext.php:96]: Maybe `rm setup.sql` on AfterSuite?

File: stepup/tests/behat/features/replay.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Feature: A replay is performed on Middleware
+  In order to replay an event stream
+  On the command line
+  I expect tha last event to be reflected in the data set
```
Comment (johanib) [inline stepup/tests/behat/features/replay.feature:4]:
```suggestion
  I expect the last event to be reflected in the data set
```

**Review by johanib — APPROVED** [review]
lgtm

File: stepup/tests/behat/features/bootstrap/FeatureContext.php (hunk 1 above)
Comment (pablothedude) [reply stepup/tests/behat/features/bootstrap/FeatureContext.php:96]: Personally I like it to leave the file afterwards in order to quickly see if it worked as expected.

**Pull request merged**